packages/repo-tools/src/release/changelog-renderer.js

```diff
@@ -1,6 +1,7 @@
 // @ts-check
 const {
   default: defaultChangelogRenderer,
+  // eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
cc @JamesHenry - it would be nice if there was a more sanctioned path here.

Maybe the package should add `nx` to the `package.json`?

Reply:
Good catch, I was confusing myself earlier. Added to repo-tools/devDeps 👍🏻